src/PrestaShopBundle/Entity/Repository/RequestSqlRepository.php

```diff
@@ -97,10 +106,19 @@ public function getSearchQueryBuilder(SearchCriteriaInterface $searchCriteria =
     }
 
     /**
-     * {@inheritdoc}
+     * Get query that counts grid rows.
+     *
+     * @param SearchCriteriaInterface $searchCriteria
+     *
+     * @deprecated deprecated since 1.7.8.0
+     * @see RequestSqlQueryBuilder::getCountQueryBuilder
+     *
+     * @return QueryBuilder
      */
     public function getCountQueryBuilder(SearchCriteriaInterface $searchCriteria = null)
     {
+        @trigger_error(sprintf('The "%s()" method is deprecated since 1.7.8.0', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error(
            sprintf(
                'The "%s()" method is deprecated since 1.7.8.0. Use %s instead.', 
                __METHOD__, 
                RequestSqlQueryBuilder::class . '::getCountQueryBuilder()'
            ), 
            E_USER_DEPRECATED
        );
```